lib/openxml/has_properties.rb

```diff
@@ -142,6 +157,17 @@ def property_xml(xml)
       end
     end
 
+    def build_required_properties
+      required_properties.each do |prop|
+        public_send(:"#{prop}=", default_property_value_for(prop)) if respond_to? :"#{prop}="
+        public_send(:"#{prop}")
```

Comment:
Does `public_send(:"#{prop}")` work because it assumes that the property name will construct and memoize the property?

Reply:
Yep. And since that's part of the `property` method's declaration above, I don't feel hacky using it in that way. 😉